test/integration/tests/utils/getCookie.ts

```diff
@@ -91,9 +91,9 @@ export async function xxsrftoken()  {
 
 function getPuppeteerLaunchOptions(url) {
   const puppeteerOption = { ignoreHTTPSErrors: true, headless: true, args: [] };
-  if (!url.includes('manage-org.')) {
-    puppeteerOption.args.push('--proxy-server=http://proxyout.reform.hmcts.net:8080');
-  }
+ // if (!url.includes('manage-org.')) {
```

Comment:
Should these lines be deleted rather than commented out?

Reply:
As per PR description: one test has been temporarily commented out by QA.

Reply:
Ah! My bad; I missed that bit of the description.